Project.toml

```diff
@@ -43,7 +45,6 @@ cuDNN = "02a925ec-e4fe-4b08-9a7e-0d78e3d38ccd"
 
 [compat]
 julia = "1"
-Lux = "0.5.68"
```

Comment:
is this not needed anymore?

Reply:
initially this change was to avoid dependency clashes with other dependencies. Namely SciML packages have been pinning Lux to this version.

Reply:
If it works, I would leave it like this since the pinning will be enforced by downstream libraries anyway.

Reply:
I see in the CI that this indeed a problem. Newer versions of Lux do not expose `AbstractExplicitLayer` which is used in `FNO.jl`. Can you either update FNO.jl to not use that or pin Lux to this older version that still has that member?

Reply:
This looks good to me, the CI is passing.